README.md

```diff
@@ -186,6 +211,16 @@ This can be achieved via an automated deployment process with tools like
 Vagrant, chef, or Puppet, or can be set manually with cloud hosts like
 Pagodabox and Heroku.
 
+### Command Line Scripts
+
+If you need to use environment variables that you have set in your `.env` file
+in a command line script that doesn't use the Dotenv library, you can `source`
+it into your local shell session:
+
+```
+source .env
+```
+
 Contributing
```

Comment:
It is consistent with the other headlines in the file. You can't do an h3 level heading without the hash in Markdown.